src/features/offer/components/OfferIconCaptions/OfferIconCaptions.native.test.tsx

```diff
@@ -172,11 +168,12 @@ async function renderOfferIconCaptions({
   isDuo?: boolean
   isBeneficiary?: boolean
 }) {
-  server.use(
-    rest.get(env.API_BASE_URL + '/native/v1/me', (_req, res, ctx) =>
-      res.once(ctx.status(200), ctx.json({ ...userProfileAPIResponse, isBeneficiary }))
-    )
-  )
+  mockUseAuthContext.mockReturnValue({
```

Comment:
Ça fait un mockReturnValue quand on appelle renderOfferIconCaptions dans un it 😬 la règle ESLint capte pas, mais c'est pas dingue comme pratique ⚠️